src/ducks/subject.js

```diff
@@ -166,51 +166,40 @@ const selectSubjectSet = (id) => {
     meant to be the first fetchSubject of the user's session. Hence, the Subject
     will ONLY be fetched IF no Subject has previously been fetched.
  */
-const fetchSubject = (id = config.zooniverseLinks.workflowId, initialFetch = false) => {
+const fetchSubject = (initialFetch = false) => {
   return (dispatch, getState) => {
     if (initialFetch && getState().subject.status !== SUBJECT_STATUS.IDLE) return;
+    const workflow_id = getState().workflow.id;
 
     dispatch({
       type: FETCH_SUBJECT,
     });
 
-    //BETA_ONLY
-    //----------------
-    let subjectQuery = {
-      workflow_id: id,
-      subject_set_id: config.zooniverseLinks.betaSubjectSet,
-    };
-
+    // BETA_ONLY
     //----------------
+    let subjectQuery = { workflow_id };
 
-    //Removed for //BETA_ONLY
     //----------------
-    /*
     let randomSubjectSet;
     const workflow = getState().workflow.data;
     if (workflow && workflow.links.subject_sets.length) {
       const linkedSets = workflow.links.subject_sets;
-      randomSubjectSet = linkedSets[Math.floor(Math.random()*linkedSets.length)];
+      randomSubjectSet = linkedSets[Math.floor(Math.random() * linkedSets.length)];
```

Comment:
This is where the second subject set is introduced. When constructing a query, a random linked subject set is chosen. In this use case, we would have both the first and second beta subject sets linked to the workflow, and one would be chosen at random.

Reply:
Noted; thanks for pointing this out.